src/pg/test/expected/06_segmentation_test.out

```diff
@@ -0,0 +1,30 @@
+\pset format unaligned
+\set ECHO all
+\i test/fixtures/ml_values.sql
+SET client_min_messages TO WARNING;
+\set ECHO none
+_cdb_random_seeds
+
+(1 row)
+prediction
+4.5656517130822492
```

Comment:
should these be set to do percent difference instead of exact equality since there's an element of randomness in the calculations?